package.json

```diff
@@ -68,7 +68,7 @@
     "formidable": "^1.1.1",
     "fs": "0.0.2",
     "govuk-elements-sass": "^3.1.2",
-    "govuk_template_jinja": "^0.23.0",
+    "govuk_template_jinja": "^0.24.0",
```

Comment:
any changes in the template that need to be applied?

Reply:
I reviewed it earlier. all the changes are just fonts and some new css classes